sample_rules/simple0_delay.yaml

```diff
@@ -0,0 +1,5 @@
+- condition: transmission_gear == 'reverse'
+  emit:
+    signal: car.backup
+    value: true
+    delay: 100
```

Comment:
Change this delay to 1000 so it's readily observed when loading this config file manually; 100ms delays are both hard to notice on its own and easily lost in any incidental system load.